packages/babel-preset-stage-0/src/index.js

```diff
@@ -2,7 +2,7 @@ import { declare } from "@babel/helper-plugin-utils";
 import presetStage1 from "@babel/preset-stage-1";
 
 import transformFunctionBind from "@babel/plugin-proposal-function-bind";
-import { proposals } from "@babel/plugin-proposal-pipeline-operator";
+import { proposals } from "@babel/plugin-syntax-pipeline-operator";
```

Comment:
FYI, not that it matters since we'll land #8293 soon, but I noticed that this is also missing from https://github.com/valtech-nyc/babel/blob/0256b8739954f803959d45c298cf79163e4a68f5/packages/babel-preset-stage-0/package.json